### ISSUE TITLE
Report 1 Command failed: /opt/homebrew/bin/swiftlint --quiet --reporter json --format --fixError: No lintable files found at paths: ''

### ISSUE THREAD

**Issue by talktosalvador** [vknabel/vscode-swiftlint#54]
- add the parameter `--fix` to fix it automatically on save, but the extension often has problems, is there another better way to automate it?

![image](https://user-images.githubusercontent.com/81868766/204573216-a4b9e817-0351-422f-bfe0-9575357cc887.png)

- the error only appears when this option is enabled

![image](https://user-images.githubusercontent.com/81868766/204572452-20aaaadf-70fb-495a-aaae-38f69e4890a1.png)

ERROR:
```
Error: Command failed: /opt/homebrew/bin/swiftlint --quiet --reporter json --format --fix
Error: No lintable files found at paths: ''

	at ChildProcess.exithandler (node:child_process:406:12)
	at ChildProcess.emit (node:events:390:28)
	at maybeClose (node:internal/child_process:1064:16)
	at Socket.<anonymous> (node:internal/child_process:450:11)
	at Socket.emit (node:events:390:28)
	at Pipe.<anonymous> (node:net:687:12)
os: darwin
```

**Comment by vknabel** [comment]
Hi @talktosalvador,

The problem with `--fix` is that behaves differently than without.
To automatically fix all issues within a document on save, add the following to your `.vscode/settings.json`:

```json
{
  "editor.codeActionsOnSave": {
    "source.fixAll": true
  }
}
```

Regarding the error you received: this should be fixed in the next release.

**Comment by vknabel** [comment]
Has been fixed in [1.7.3](https://github.com/vknabel/vscode-swiftlint/releases/tag/1.7.3).
If your issue has not been fixed yet, feel free to reopen.
